docs/installation.md

```diff
@@ -62,17 +56,26 @@ For this rule to set up the correct permissions for your receiver
 you will then need to either physically remove the receiver and
 re-insert it or reboot your computer.
 
-Then solaar can be run without using sudo.
 
+## Running from the Download Directories
 
-### Installing Solaar
+If Solaar's udev rule is installed,
+you can just go to the solaar directory and run `bin/solaar` for the GUI
+or `bin/solaar <command> <arguments>` for the CLI.
+
+Otherwise you will need to run Solaar as root via
+and `sudo bin/solaar` for the GUI
+or `sudo bin/solaar <command> <arguments>` for the CLI.
+
+
+## Installing Solaar
 
 Python programs are usually installed using [pip][pip].
 The pip instructions for solaar are in `setup.py`, the standard place to put such instructions.
 
 To install solaar for yourself only run `pip install --user .` from the solaar directory.
-This tells pip to install into your `.local` directory. You can then run solaar as 
- `~/.local/bin/solaar`.
+This tells pip to install into your `.local` directory, but does not install Solaar's udev rule.
```

Comment:
Is there a way, I guess as extra step as root, to install the udev rule for this local pip installation? So that after that root is not needed anymore?

Reply:
That's elsewhere in the document, in the section "Installing Solaar's udev Rule"

Reply:
Maybe at a note like ”see section above to install the rule” ?

Reply:
Yeah, coming momentarily.